### DISCUSSION TITLE
What are the specific differences between the three project types?

### DISCUSSION THREAD

**Discussion by atrauzzi** [AvaloniaUI/Avalonia.Templates#268] [Q&A] [answered]
I see ".NET App", ".NET MVVM App" and "Cross Platform Application"

I would have thought there'd be only one type of Avalonia project and then it simply would support being compiled to different targets?

When creating a new project, how can I be informed of what each choice entails?  Can I convert between the project types?  What are the specific implications of the choice?

**Comment by maxkatz6** [comment, marked as answer]
It's not really "types" of the projects. But bootstrap templates with some code pre-written. 

`.NET App` - a single executable project. In current .NET world it can only be run on Desktop OSes, like windows, macos or linux.
`.NET MVVM App` - the same as above, a single executable project for Desktop, but with MVVM framework pre-setup.

`XPlat Application` - probably a confusing name, but it's a template with multiple executable projects for different platforms (including mobile and browser), with shared code extracted.
